### PR TITLE
i am create for loop

### PR DESCRIPTION
**Pull request by ajitjadhav0505**

### CODE REVIEW

File: looping,js [hunk 1]
```diff
@@ -0,0 +1 @@
+for(i=0)
```
Comment (Deepakgadhave9527) [inline looping,js:1]: for loop is not working

Comment (ajitjadhav0505) [inline looping,js:1]: ok i will do

**Comment by ajitjadhav0505** [comment]
not done

File: looping,js [hunk 2]
```diff
@@ -1 +1,3 @@
-for(i=0)
+for(i=0;i<10;i++){
```
[the remark this answers is not in the document]
Comment (Deepakgadhave9527) [reply looping,js:1]: not  solve

**Comment by Deepakgadhave9527** [comment]
goood to done

**Pull request merged**